cloudposse/terraform-aws-elastic-beanstalk-environment
# DEVOPS-229: Fix boolean values for new settings and add description variable

## Problem

AWS expects `"true"` or `"false"` and not the interpolated value from TF.

## Solution

- Used conditionals on new values to ensure `"true"` or `"false"` are sent
- Added a new variable for environment description

## Testing

- Update `examples/complete/main.tf`'s source to `../../`
- Plan it
- Verify the plan completes successfully
